prow/jobs/runtime-watcher/runtime-watcher.yaml

```diff
@@ -149,6 +149,79 @@ presubmits: # runs on PRs
             effect: NoSchedule
         nodeSelector:
             dedicated: "high-cpu"
+    - name: pre-main-runtime-watcher-listener-lint
```

Comment:
We had small discussion internally, that most of ProwJobs' names are way too long. We agreed upon shortening the name from `pre-main-...` to just `pull-` Can you consider changing the names accordingly? Also you don't need to include the repository name. I believe `pull-listener-lint` would be enough to distinguish this job from the others.

Same applies to the other jobs.